README.md

```diff
@@ -17,12 +17,20 @@ A drop-in replacement for the standard Zipkin HTTP collector that writes to the
 Stackdriver Trace service.
 
 ### Configuration
+Typically, sampling is done by the Tracer prior to sending data to the Collector, so the Collector
```

Comment:
shouldn't this be in the table as a description of COLLECTOR_SAMPLE_RATE? or maybe a section called sampling?